lib/resources/example_json/r4_examples_observation.rb

```diff
@@ -2,6 +2,177 @@
 
 module Cerner
   module Resources
+    # TODO: All; Missing mappings
```

Comment:
TODO?

Reply:
Thanks for pointing it out, I think it got added during the rebase.
Removed it https://github.com/cerner/fhir.cerner.com/pull/442/commits/6223f7e858de14670d4a1bb7ff11e7d6e2ca67af#diff-cd059d38e44f892deed0ab35bef7831cL5